client/my-sites/plugins/plugin-list-header/index.jsx

```diff
@@ -344,41 +349,77 @@ export default React.createClass( {
 				</DropdownItem>
 
 				<DropdownItem key="plugin__actions_disable_autoupdate"
 						disabled={ ! this.canUpdatePlugins() }
 						onClick={ this.props.unsetAutoupdateSelected }>
-					{ this.translate( 'Disable Autoupdates' ) }
+					{ translate( 'Disable Autoupdates' ) }
 				</DropdownItem>
 
 				<DropdownSeparator key="plugin__actions_separator_3" />
 
-				<DropdownItem key="plugin__actions_remove"
-						className="plugin-list-header__actions_remove_item"
-						disabled={ ! needsRemoveButton }
-						onClick={ this.props.removePluginNotice } >
-					{ this.translate( 'Remove' ) }
+				<DropdownItem
+					key="plugin__actions_remove"
+					className="plugin-list-header__actions-remove-item"
+					disabled={ ! needsRemoveButton }
+					onClick={ this.props.removePluginNotice } >
+					{ translate( 'Remove' ) }
 				</DropdownItem>
 			</SelectDropdown>
 		);
-	},
+	}
 
 	render() {
-		const sectionClasses = classNames( 'plugin-list-header', { 'is-bulk-editing': this.props.isBulkManagementActive, 'is-action-bar-visible': this.state.actionBarVisible } );
+		const { label, selected, plugins, isBulkManagementActive } = this.props;
+		const sectionClasses = classNames( {
+			'plugin-list-header': true,
+			'is-bulk-editing': isBulkManagementActive,
+			'is-action-bar-visible': this.state.actionBarVisible
+		} );
 		return (
-			<SectionHeader label={ this.props.label } className={ sectionClasses }>
+			<SectionHeader label={ label } className={ sectionClasses }>
 				{
-					this.props.isBulkManagementActive &&
+					isBulkManagementActive &&
 						<BulkSelect key="plugin-list-header__bulk-select"
-							totalElements={ this.props.plugins.length }
-							selectedElements={ this.props.selected.length }
+							totalElements={ plugins.length }
+							selectedElements={ selected.length }
 							onToggle={ this.unselectOrSelectAll } />
 				}
 				{ this.renderCurrentActionDropdown() }
 				{ this.renderCurrentActionButtons() }
 			</SectionHeader>
 		);
 	}
-} );
+}
+
+PluginsListHeader.propTypes = {
+	label: PropTypes.string,
+	isBulkManagementActive: PropTypes.bool,
+	toggleBulkManagement: PropTypes.func.isRequired,
+	updateAllPlugins: PropTypes.func.isRequired,
+	updateSelected: PropTypes.func.isRequired,
+	haveUpdatesSelected: PropTypes.bool,
+	pluginUpdateCount: PropTypes.number.isRequired,
+	activateSelected: PropTypes.func.isRequired,
+	deactiveAndDisconnectSelected: PropTypes.func.isRequired,
+	deactivateSelected: PropTypes.func.isRequired,
+	setAutoupdateSelected: PropTypes.func.isRequired,
+	setSelectionState: PropTypes.func.isRequired,
+	unsetAutoupdateSelected: PropTypes.func.isRequired,
+	removePluginNotice: PropTypes.func.isRequired,
+	haveActiveSelected: PropTypes.bool,
+	haveInactiveSelected: PropTypes.bool,
+	bulkManagement: PropTypes.bool,
+	selectedSiteSlug: PropTypes.string,
+	plugins: PropTypes.array.isRequired,
+	selected: PropTypes.array.isRequired
+};
+
+PluginsListHeader.defaultProps = {
```

Comment:
Any good reason to keep the `defaultProps` outside of the class? This could be brought into the class like this:

```
static defaultProps = {
	...
};
```